src/api/data.js

```diff
@@ -67,8 +68,18 @@ function getRemoteBooks() {
   .then(response => response.map(transformBooks).filter(hasISBN));
 }
 
+function getSearchRemoteBooks(searchTerm) {
+  return new Promise((resolve) => {
+     window.setTimeout(
+       () => {
+         resolve(bookData);
+       }, 100);
+   });
+}
+
```

Comment:
Fake api request - should be replaced with a proper api request the preforms a search